src/header/mod.rs

```diff
@@ -433,7 +439,7 @@ impl fmt::Show for Item {
             None => match self.raw {
                 Some(ref raw) => {
                     for part in raw.iter() {
-                        try!(fmt.write(part.as_slice()));
+                        try!(write!(fmt, "{}", part.as_slice()));
```

Comment:
Just ran some of the examples to sanity check and it's fine, so I guess this works as intended.